tls/s2n_security_policies.c

```diff
@@ -1124,13 +1137,15 @@ struct s2n_security_policy_selection security_policy_selection[] = {
     { .version = "default", .security_policy = &security_policy_20240501, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "default_tls13", .security_policy = &security_policy_20240503, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
     { .version = "default_fips", .security_policy = &security_policy_20240502, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
+    { .version = "default_pq", .security_policy = &security_policy_20240730, .ecc_extension_required = 0, .pq_kem_extension_required = 0 },
```

Comment:
Do we want a new `default_*` that we have to deal with?

Reply:
I don't think we want to deal with it, but I unfortunately do think we need it. Otherwise customers who aren't particularly interested in anything other than enabling PQ are going to have to parse through the security policy tables to try to figure out which one they want.

At worst, it'd have the same deprecation strategy as "default_tls13". If "default" ever includes PQ, we can just alias "default_pq" to "default".